src/main/application-update/check-for-updates-tray-item.injectable.ts

```diff
@@ -59,9 +59,11 @@ const checkForUpdatesTrayItemInjectable = getInjectable({
 
       click: pipeline(
         async () => {
-          await processCheckingForUpdates("tray");
+          const { updateIsReadyToBeInstalled } = await processCheckingForUpdates("tray");
 
-          await showApplicationWindow();
+          if (updateIsReadyToBeInstalled) {
+            await showApplicationWindow();
```

Comment:
There is no point to show just the application window after user intentionally checked for updates. We should show message at least or dedicated page says `New update is available` with some update info and an action button ("Update now"). Could be done as a separate PR though.

Reply:
Hmm. I'd say it's not that bad, since first thing you see when the application opens in this case is the "Update" button in the top bar. :)